packages/builder/src/builder.js

```diff
@@ -659,6 +659,10 @@ export default class Builder {
     if (this.ignore.ignoreFile) {
       nuxtRestartWatch.push(this.ignore.ignoreFile)
     }
+    // If default page displayed, watch for first page creation
+    if (this._nuxtPages && this._defaultPage) {
```

Comment:
Is `_defaultPage` is enough here ?

If `_defaultPage` is true, `_nuxtPages` must be `true`.

Reply:
Yes but readability is better :)